_posts/2021-01-12-updating-foreman-inventory-with-system-facts.md

```diff
@@ -0,0 +1,323 @@
+---
+layout: post
+title: Updating Foreman inventory with system facts
+date: 2021-01-12 17:00:00
+author: Marek Hulan
+tags:
+- foreman
+- community
+- tutorial
+- howto
+- beginners
+---
+
+This is a continuation of [How to start with Foreman post](https://theforeman.org/2020/12/how-to-start-with-foreman.html).
+Today we'll look at how the inventory can be kept up to date by receiving facts from hosts regularly. Then we'll take a look
+where facts can be used. We'll illustrate the configuration on one config management system, but the same concept is applicable
+to other facts sources. Links to the related documentation will be provided.
+
+<!--more-->
+
+## Automatically update the inventory from configuration management
+
+Having to keep a big inventory up to date manually would be a lot of work. Foreman
+can receive updates from other tools used to manage the infrastructure and reflect
+them in the database. Also new Hosts can be created this way. If you use some
+configuration management system or you use a subscription manager (mostly RPM
+based distributions), you can configure it to forward data about managed hosts to Foreman. If the host does not exist in Foreman yet, it can be automatically created. If it exists, the record is updated accordingly.
```

Comment:
Can or will be automatically created? Can you prevent automatic creation if the data is forwarded to Foreman?

Reply:
*can*, based on the settings described below, I updated the update sentece, so it's also *can*